### PR TITLE
add full stack tekton on openshift

### PR DESCRIPTION
**Pull request by adrian555**

### CODE REVIEW

File: OpenShift/manifests/README.md [hunk 1]
```diff
@@ -52,27 +52,36 @@ There are single-user and multi-user options to deploy Kubeflow. Choose one that
 
 * Single-user
 
-  Use this [KfDef configuration](./kfctl_tekton_openshift_minimal.v1.1.0.yaml) file to deploy the minimal required components for single-user Kubeflow with Tekton pipeline.
+  Use this [KfDef configuration](./kfctl_openshift_tekton.v1.1.0.yaml) file to deploy the minimal required components for single-user Kubeflow with Tekton pipeline.
```
Comment (animeshsingh) [inline OpenShift/manifests/README.md]:
```suggestion
  Use this [KfDef configuration](./kfctl_openshift_tekton.v1.1.0.yaml) file to deploy Kubeflow with Kubeflow Pipelines-Tekton combination. Please note that this manifest is derived from OpenDataHub manifest for Kubeflow, and we have customized Kubeflow Pipelines with Tekton backend. Additionally this deployment doest not cover multi-user auth/authz scenario.
```

Comment (animeshsingh) [reply OpenShift/manifests/README.md]: Additionally is it version 1.1? Should be KF 1.0?

Comment (adrian555) [reply OpenShift/manifests/README.md]: No, this is based on Kubeflow 1.1.0. Only copied the relative openshift-scc and openshift layers for several applications from odh. So all are in Kustomize v3.

Comment (animeshsingh) [reply OpenShift/manifests/README.md]: hmmm - so KF 1.1 can run on OpenShift?

Comment (adrian555) [reply OpenShift/manifests/README.md]: Did not try KFP. But for Katib, Seldon etc I am using their manifests in v1.1-branch, i.e. I create an openshift kustomize v3 stack similar to IBM's one.

**Pull request closed without merging**